print-udp.c

```diff
@@ -646,6 +646,8 @@ udp_print(netdissect_options *ndo, register const u_char *bp, u_int length,
 			geneve_print(ndo, (const u_char *)(up + 1), length);
 		else if (IS_SRC_OR_DST_PORT(LISP_CONTROL_PORT))
 			lisp_print(ndo, (const u_char *)(up + 1), length);
+		else if (dport == GUE_PORT)
```

Comment:
It seems from the spec that the reponse from the middlebox to the faraway host would have UDP ports the other way round (from GUE_PORT to any other port), should it be accounted here as well?